roles/junit2json/defaults/main.yml

```diff
@@ -0,0 +1,13 @@
+# SPDX-License-Identifier: Apache-2.0
+---
+# defaults file for role redhatci.ocp.junit2json
+
+# safeguard global variable:
+global_json_reports_list: []
+
+# defaults:
+junit2json_input_merged_report: ''
```

Comment:
already left a note up in README, according to the docs this is only required if `do_merge` is true (the default in line 13) so either you make this a sane default, or remove the default and make it required because right now it is bad UX